resources/language/Croatian/strings.po

```diff
@@ -0,0 +1,270 @@
+msgstr "Ocjeni na Netflixu"
+
+msgctxt "#30020"
+msgid "Remove from 'My list'"
+msgstr "Ukloni s 'Moj popis'"
+
+msgctxt "#30021"
+msgid "Add to 'My list'"
+msgstr "Dodaj na 'Moj popis'"
+
+msgctxt "#30022"
+msgid "(between 0 & 10)"
+msgstr "(između 0 i 10)"
+
+msgctxt "#30023"
+msgid "Expert"
+msgstr "Stručno"
+
+msgctxt "#30024"
+msgid "SSL verification"
+msgstr "SSL ovjera"
+
+msgctxt "#30025"
+msgid "Library"
+msgstr "Videoteka"
+
+msgctxt "#30026"
+msgid "Enable custom library folder"
+msgstr "Omogući prilagođenu mapu videoteke"
+
+msgctxt "#30027"
+msgid "Custom library path"
+msgstr "Prilagođena putanja videoteke"
+
+msgctxt "#30028"
+msgid "Playback error"
+msgstr "Greška reprodukcije"
+
+msgctxt "#30029"
+msgid "Missing Inputstream addon"
+msgstr "Nedostaje dodatak ulaznog strujanja"
+
+msgctxt "#30030"
+msgid "Remove from library"
+msgstr "Ukloni iz videoteke"
+
+msgctxt "#30031"
+msgid "Change library title"
+msgstr "Promijeni naslov videoteke"
+
+msgctxt "#30032"
+msgid "Tracking"
+msgstr "Praćenje"
+
+msgctxt "#30033"
+msgid "Use Dolby Sound"
+msgstr "Koristi Dolby zvuk"
+
+msgctxt "#30034"
+msgid "ESN (set automatically, can be changed manually)"
+msgstr "ESN (postavi automatski, može se promijeniti ručno)"
+
+msgctxt "#30035"
+msgid "Inputstream Addon Settings..."
+msgstr "Postavke dodatka ulaznog strujanja..."
+
+msgctxt "#30036"
+msgid "Always use the original title on export"
+msgstr "Uvijek koristi izvorni naslov pri izvozu"
+
+msgctxt "#30037"
+msgid "Views"
+msgstr "Prikazi"
+
+msgctxt "#30038"
+msgid "Enable custom views"
+msgstr "Omogući prilagođene prikaze"
+
+msgctxt "#30039"
+msgid "View for folders"
+msgstr "Prikaz mapa"
+
+msgctxt "#30040"
+msgid "View for movies"
+msgstr "Prikaz filmova"
+
+msgctxt "#30041"
+msgid "View for shows"
+msgstr "Prikaz serija"
+
+msgctxt "#30042"
+msgid "View for seasons"
+msgstr "Prikaz sezona"
+
+msgctxt "#30043"
+msgid "View for episodes"
+msgstr "Prikaz epizoda"
+
+msgctxt "#30044"
+msgid "View for profiles"
+msgstr "Prikaz profila"
+
+msgctxt "#30045"
+msgid "[COLOR cyan][B]-- NEXT PAGE --[/B][/COLOR]"
+msgstr "[COLOR cyan][B]-- SLJEDEĆA STRANICA --[/B][/COLOR]"
+
+msgctxt "#30046"
+msgid "Inputstream addon is not enabled"
+msgstr "Dodatak ulaznog strujanja nije omogućen"
+
+msgctxt "#30047"
+msgid "Finally remove?"
+msgstr "Konačno ukloni?"
+
+msgctxt "#30048"
+msgid "Exported"
+msgstr "Izvezeno"
+
+msgctxt "#30049"
+msgid "Update DB"
+msgstr "Nadopuni bazu podataka"
+
+msgctxt "#30050"
+msgid "Update successful"
+msgstr "Nadopuna uspješna"
+
+msgctxt "#30051"
+msgid "Request Error"
+msgstr "Greška zahtjeva"
+
+msgctxt "#30052"
+msgid "Unable to complete the request at this time"
+msgstr "Nemoguće je trenutno završavanje zahtjeva"
+
+msgctxt "#30053"
+msgid "Auto Login"
+msgstr "Automatska prijava"
+
+msgctxt "#30054"
+msgid "Enable Auto Login"
+msgstr "Omogući automatsku prijavu"
+
+msgctxt "#30055"
+msgid "Profile"
+msgstr "Profil"
+
+msgctxt "#30056"
+msgid "ID"
+msgstr "ID"
```

Comment:
Can you remove that `msgstr` contents, if `msgid` & `msgstr` are the same, `msgstr` should be empty